src/easy_config/contrib/wtforms.py

```diff
@@ -0,0 +1,29 @@
+"""Test the Flask-WTF wrapper."""
+
+import dataclasses
+from typing import Type
+
+from easy_config import EasyConfig
+from flask_wtf import FlaskForm
+from wtforms import BooleanField, IntegerField, StringField
+from wtforms.validators import DataRequired
+
+type_to_field = {
+    str: StringField,
+    int: IntegerField,
+    bool: BooleanField,
+}
```

Comment:
Can WTForms only handle these three datatypes?

Reply:
it does other things.. this is not anywhere near complete (but at least is a proof-of-concept)